maintainers/maintainer-list.nix

```diff
@@ -1711,6 +1705,12 @@
     githubId = 2245737;
     name = "Christopher Mark Poole";
   };
+  chvp = {
```

Comment:
Please change the commit message to ``maintainers: rename charvp to chvp``.

Reply:
Done.